### ISSUE TITLE
Perceptual Loss

### ISSUE THREAD

**Issue by zhangqianhui** [taconite/arah-release#26]
How to use perceptual loss, is patch sampling necessary for using perceptual loss ?

**Comment by taconite** [comment]
Yes, to use perceptual loss you need to sample rectangular patches from the images (usually 32x32). We did not use perceptual loss in our work but this should be feasible by modifying the dataloader.
